ros/src/computing/perception/detection/packages/cv_tracker/nodes/range_fusion/range_fusion.cpp

```diff
@@ -88,10 +88,32 @@ int main(int argc, char **argv)
 	ros::init(argc, argv, "range_fusion");
 
 	ros::NodeHandle n;
-
-	ros::Subscriber image_obj_sub = n.subscribe("image_obj", 1, DetectedObjectsCallback);
+	ros::NodeHandle private_nh("~");
+
+	std::string image_topic;
+	std::string points_topic;
+	if (private_nh.getParam("image_node", image_topic))
+    	{
+        	ROS_INFO("Setting image node to %s", image_topic.c_str());
+    	}
+	else
+	{
+		ROS_INFO("No image node received, defaulting to image_obj, you can use _image_node:=YOUR_TOPIC");
+		image_topic = "/image_obj";
+	}
+	if (private_nh.getParam("points_node", points_topic))
+    	{
+        	ROS_INFO("Setting points node to %s", points_topic.c_str());
+    	}
+	else
+	{
+		ROS_INFO("No points node received, defaulting to vscan_image, you can use _points_node:=YOUR_TOPIC");
+		points_topic = "/vscan_image";
```

Comment:
@syohex ,
Thanks for notification.
I think `image_obj` should be relative topic path.
But with regard to `vscan_image`, it's OK to be absolute one because `vscan_image` doesn't belongs to any namespace.
I will confirm the behavior of this branch.